src/widget/gui.cpp

```diff
@@ -322,25 +322,34 @@ QString GUI::passwordDialog(const QString& cancel, const QString& body)
 
 void GUI::_clearContacts()
 {
-    Nexus::getDesktopGUI()->clearContactsList();
+    Widget* w = Nexus::getDesktopGUI();
+    if (w)
+        w->clearContactsList();
```

Comment:
1) It's coding style. What the reason to have coding style if we will not stick it?
2) I'm absolutely [agree](https://github.com/qTox/qTox/pull/3839#issuecomment-277912589) with you about K&R braces. Can you vote on #3839 ?